libs/MeshKernel/include/MeshKernel/Definitions.hpp

```diff
@@ -97,8 +97,10 @@ namespace meshkernel
     /// @brief The possible types of the values to be interpolated in the gridded sample
     enum class InterpolationValues
```

Comment:
I would:
- rename this to enum class InterpolationDataTypes
- capitalize the first letter and remove the suffix Type of each enumeration
- Have int type before double type (more coherent with existing enum)
```
enum class InterpolationDataTypes {
  Short = 0,
  Float = 1,
  Int = 2,  // moved up
  Double = 3
};
```